xarray/core/dataset.py

```diff
@@ -2384,6 +2384,34 @@ def from_dataframe(cls, dataframe):
             obj[name] = (dims, data)
         return obj
 
+    def to_dask_dataframe(self, set_index=True):
```

Comment:
I don't think we want to default to `set_index=True`, since xarray focuses on handling N-dimensional data, so data known ahead of time to be 1D is somewhat rare.